src/Angor/Client/Pages/Signatures.razor

```diff
@@ -172,43 +148,57 @@
 
                     pendingSignature.AmountToInvest = investorTrx.Outputs.AsIndexedOutputs().Skip(2).Take(investorTrx.Outputs.Count - 3) //Todo get the actual outputs with taproot type
                         .Sum(_ => _.TxOut.Value);
-                    
-                    StateHasChanged();
                 }
                 catch (Exception e)
                 {
                     Console.WriteLine(e);
-                    Console.WriteLine(pendingSignature.TransactionHex);
+                    _Logger.LogDebug(pendingSignature.TransactionHex);
```

Comment:
probably not logging the correct thing here?

Reply:
It's only for debugging, I will remove it..